### DISCUSSION TITLE
Add a data base

### DISCUSSION THREAD

**Discussion by AdarianHeart89** [bucanero/pkgi-ps3#66] [General]
I am looking at adding a database to my config.txt file but the one I want to use does not have a tsv but it allows me to download a csv. How would I go adding it to the config.txt or should I just add the csv to the pkgi folder on my PS3?

**Comment by bucanero** [comment]
unless the file complies with the pkgi database format, you'll need to either generate a compatible file, or create a `dbformat.txt` file to describe your custom file.

See the Readme about databases and custom formats.
